ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/ThreadNet/Network.hs

```diff
@@ -607,8 +611,9 @@ runThreadNetwork ThreadNetworkArgs
         , cdbBlockchainTime       = btime
         , cdbAddHdrEnv            = nodeAddHeaderEnvelope (Proxy @blk)
         , cdbImmDbCacheConfig     = Index.CacheConfig 2 60
-          -- Misc
-        , cdbTracer               = instrumentationTracer <> nullDebugTracer
+        -- Misc
+        , cdbTracer               = instrumentationTracer <>
+                                    showTracing (decorateId coreNodeId debugTracer)
```

Comment:
TODO The test suite not generate any traces by default, so this line shouldn't be merged as-is. (Let me know if you disagree.)

Reply:
Yes, this was a rebase artifact, so not sure how it got in anyway.